### ISSUE TITLE
Fluid storage economy?

### ISSUE THREAD

**Issue by PONYRYDE** [raoulvdberge/refinedstorage#918]
64k Storage Part =  27 * 1k Storage Part = 2.3704 times storage
512k Fluid Storage Part  = 27 * 64k Fluid Storage Part = 0.2963 times storage

Fluid Storage Parts/Disks/Blocks should increase four times each tier to match item storage. Re-balancing to 16/64/256/1024 would resolve this and make RS fluid storage more attractive.

Since we are here.
It would also be nice to be able to import and directly export fluids without requiring intermediary storage. Currently (at least in 1.10.2) I can't import unless there is fluid storage available, not even with an interface that has an internal buffer.

**Comment by raoulvdberge** [comment]
Fluid storage tiering doesn't follow any logic, it's mostly those amounts for balancing reasons.

You gave a 16k fluid storage as an example, the reason I didn't go for something like that is because 16k can only store 16 buckets of fluid. That isn't enough for the first tier.

**Comment by raoulvdberge** [comment]
Also, changing the existing tiering isn't really possible because they are already existing disks with the old tiers.

**Comment by PONYRYDE** [comment]
Given how cheap tier 1 is 16B seems reasonable when compared to more than 27x the cost for only 8x more storage with tier 4. 16B is enough to auto craft with but I don't consider 512B to be a lot of storage, it's because I went to craft a bunch of them I realized how expensive they are when compared to being able to store 64000 blocks in tier 4 item storage. I would of suggested 32/128/512/2048 but that seems a little OP at the higher end, which is I guess your point. Maybe a config option would be possible?
